codeforboston/police-data-trust
# Refactor and tests

Closes #252 

How to test refactor:
- <code>npm run dev</code>
- Navigate to /officer/0
- Confirm the officer page displays data
- Navigate to different officers using different ids (ex. /officer/12, /officer/1) [0..99] for now

How to test tests:
- <code>npm test</code>
